recipes/kaleido-core/bld.bat

```diff
@@ -0,0 +1,11 @@
+set APP_DIR=%PREFIX%\Library\bin\KaleidoApp
+set LAUNCH_SCRIPT=%PREFIX%\Library\bin\kaleido.cmd
+set BIN_LOCATION=%APP_DIR%/kaleido.cmd
+
+mkdir %APP_DIR%
+xcopy * %APP_DIR% /E/H
```

Comment:
This should have an exit condition after it: eg https://conda-forge.org/docs/maintainer/knowledge_base.html#simple-cmake-based-bld-bat

Reply:
Thanks, is https://github.com/conda-forge/staged-recipes/pull/12747/commits/7c6fe755f044894c2fdcc7b2782ac4ca3d6d3ec2 what you had in mind?